### DISCUSSION TITLE
Can I use nativewind with react native paper?

### DISCUSSION THREAD

**Discussion by WinKoKoNaing** [nativewind/nativewind#251] [Q&A] [answered]
I used both in my project, I want to know it is good or bad.

**Comment by marklawlor** [comment, marked as answer]
NativeWind works with any component that accepts a `style` prop.

I can't answer "good or bad", as both libraries have very different goals. NativeWind is a utility styling library, while React Native Paper is a component library.

If you don't quite understand the differences, there's a lot of articles online talking about Material UI (a component library) and Tailwind CSS and how they relate to each other.

**Reply by WinKoKoNaing** [reply]
Thanks for your reply @marklawlor

**Comment by danstepanov** [comment]
Closing this discussion, if this is still relevant, feel free to open a new one.
